azuredevops/internal/service/core/resource_team.go

```diff
@@ -0,0 +1,558 @@
+package core
```

Comment:
Because the mentioned functions are only used within the scope of teams management and I don't expect that we have some sort of re-usage like with the helper functions in the `permission/utils` package, I'd like to keep the functions where they are located right now. But I see the point: when I look at the acceptance test files, that could be compiled and executed separately, this would definitely be an issue if (helper) functions are only located in one file that is not part of the compile unit. But as already said: I don't see that gonna happen here.